rclpy/rclpy/node.py

```diff
@@ -761,6 +763,25 @@ def _set_parameters_atomically(
 
         return result
 
+    def add_on_set_paramters_callback(
+        self,
+        callback: Callable[[List[Parameter]], SetParametersResult]
+    ) -> None:
+        """Add the callback to list."""
+        prevCallBack = self._parameters_callback
+        prevCallBack.insert(0, callback)
```

Comment:
@suab321321 why insert into the front?

Reply:
Similar to ros2/rclcpp#722.Also cross checked with @wjwwood  code.